crates/vm/levm/src/gas_cost.rs

```diff
@@ -902,6 +904,17 @@ fn precompile(data_size: usize, static_cost: u64, dynamic_base: u64) -> Result<u
         .ok_or(OutOfGasError::GasCostOverflow)?)
 }
 
+pub fn ecpairing(groups_number: usize) -> Result<u64, VMError> {
+    let groups_number = u64::try_from(groups_number).map_err(|_| InternalError::ConversionError)?;
+
+    let groups_cost = groups_number
+        .checked_mul(34000)
+        .ok_or(OutOfGasError::GasCostOverflow)?;
+    groups_cost
+        .checked_add(45000)
```

Comment:
Quick question Maxi, could these values be put into constants? 

Would that make sense in this context?

Reply:
Done in [Add constants in gas cost](https://github.com/lambdaclass/ethrex/pull/1560/commits/ecc075ec40ed509a10ab4eb980ae28a5a6640865)!